zuul.d/edpm_build_images.yaml

```diff
@@ -26,3 +26,18 @@
       - ^ci/playbooks/edpm_build_images/.*
       - ^scenarios/centos-9/edpm_build_images_ci.yml
       - ^zuul.d/edpm_build_images.yaml
+
+- job:
+    name: periodic-edpm-build-push-images-centos-9-antelope
+    parent: cifmw-base-edpm-build-images
+    vars:
+      promote_source: podified-ci-testing
+      push_registry: quay.rdoproject.org
+      registry_namespace: podified-antelope-centos9
+      push_containers: true
+      buildah_login: true
+      registry_login_enabled: true
+      openstack_release: antelope
```

Comment:
Where do we use openstack_release?

Reply:
We need this because of condition here: https://review.rdoproject.org/r/plugins/gitiles/config/+/refs/heads/master/playbooks/crc/podman.yaml#31